tests/lib/rules/require-default-props.js

```diff
@@ -3019,30 +3019,6 @@ ruleTester.run('require-default-props', rule, {
         },
       ],
     },
-    {
-      code: `
-        function MyStatelessComponent({ foo, bar }) {
-          return <div>{foo}{bar}</div>;
-        }
-        MyStatelessComponent.propTypes = {
-          foo: PropTypes.string,
-          bar: PropTypes.string.isRequired
-        };
-        MyStatelessComponent.propTypes.baz = React.propTypes.string;
-      `,
-      errors: [
-        {
-          messageId: 'shouldHaveDefault',
-          data: { name: 'foo' },
-          line: 6,
-        },
-        {
-          messageId: 'shouldHaveDefault',
-          data: { name: 'baz' },
-          line: 9,
-        },
-      ],
-    },
```

Comment:
Duplicate of:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/require-default-props.js#L1343-L1367